Offseason Code for 2019 "Deep Space"

strimper13 opened the pull request:
Additional code changes for offseason of the 2019 robot game

strimper13 committed:
Vision W.I.P

strimper13 committed:
Resolved Slow Turn Directions

Swapped each slow turn to the opposite sign so they are correct in the nature of each of their names

Rohan-Sreenivasan commented on src/Vision/testpi.py:
```diff
@@ -300,7 +300,7 @@ def normalizeImage(pairMidpoint):
     # prints after ewww math THIS IS THE CONTOUR MIDPOINT from...
     # Contour Midpoint Scale (-1 to 1)
-    return fullPairMidpoint
+    return fullPairMidpoint 
```
You have an extra space at the end of this

msoucy replied:
![](http://4.bp.blogspot.com/-i9gjbiGZZF4/TzGWsVZnYuI/AAAAAAAAEws/MqPWmNUCfg4/s1600/spaaace.jpg)

Rohan-Sreenivasan reviewed (commented):
good code @strimper13

strimper13 committed:
Demo Driving Code

Added DemoDrive command and added command for button on copilot controller to take over drive: replaces enter button

bot190 reviewed (approved):
This looks reasonable, and I want to close out last years code.
